en/docs/services/service-concepts.md

```diff
@@ -0,0 +1,113 @@
+# Concepts
+
+Before you start creating services with Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know.  
+ 
+## Service
+
+A service in Choreo is a logical representation of a repeatable activity that has a specified outcome.
+
+## Service Sample
+
+A service sample in Choreo is a ready-to-use service that you can easily use depending on your requirement. Choreo provides a comprehensive set of service samples that you can select and use depending on your requirement. 
+
+
+## Trigger
+
+A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+
+Choreo supports the following trigger types:
+
+ - **API**: Allows you to expose an application as an API endpoint. 
+ - **Manual**: Allows you to trigger an application manually.
+ - **Schedule**: Allows to trigger an application based on a schedule that you set. You can specify the schedule in the form of a cron expression.
+   <html><div class="admonition note">
```

Comment:
Shall we use `> **Note**: ` instead of this HTML element?

Reply:
Btw do we need this section? We don't support cron based editor now.

Reply:
Removed content on the trigger concept as it is no longer applicable to services.